src/templates/certificate.html

```diff
@@ -1,90 +1,88 @@
+                    no-repeat center;
+                height: 99%;
+                background-size: contain;
+                font-size: 36px;
+            }
+        </style>
+    </head>
+    <body>
+        <div class='body'>
+            <div class='center'>
+                <div class='text-center'><div class='font-600 fs-20 py-20'>{{studentName}}</div></div>
+                <div class='d-flex jc-sp-bw'>
+                    <div class='w-45'>
+                        Student ID
+                        <div class='font-600'>{{studentID}}</div>
```

Comment:
@Ekta29yadav Can u make this `studentID` as `studentId` same being used in schema

Reply:
done